src/components/InfiniteGrids/useInfiniteGrid.ts

```diff
@@ -83,7 +83,7 @@ export const useInfiniteGrid = <
     variables: {
       ...queryVariables,
       orderBy,
-      first: sortByViews ? 100 : targetDisplayedItemsCount,
+      first: sortByViewsOrFollowers ? 100 : targetDisplayedItemsCount,
```

Comment:
I don't think we need this  `sortByViewsOrFollowers`, maybe we could just check if `additionalSortFn` is passed?  Something like:
```
first: additionalSortFn ? 100 : targetDisplayedItemsCount,
```